### PR TITLE
Only log json and add traceback logging

### PR DESCRIPTION
**Pull request by heinrisch**

### CODE REVIEW

File: tests.py [hunk 1]
```diff
@@ -25,7 +25,7 @@ def _generate_output(self, level):
         res['namespace'] = self.namespace
         res['service'] = self.service
         res['timestamp'] = self.timestamp
-        return '%s %s\n' % (level.upper(), json.dumps(res))
+        return str(json.dumps(res)) + '\n'
```
Comment (voroninman) [inline tests.py]: `json.dumps` always returns string so `str` is redundant here.

**Review by nlundbo — APPROVED** [review]

File: wrapplog.py [hunk 2]
```diff
@@ -1,22 +1,17 @@
...
 import collections
 from datetime import datetime
 
-import structlog
 from structlog.processors import JSONRenderer
 from structlog import wrap_logger, PrintLogger, PrintLoggerFactory
 
-
-# Deprecated
-def start_logging(output=None):
```
Comment (voroninman) [inline wrapplog.py]: Have you verified that we're safe to remove it? Or is it a way to enforce the removal? :)

**Comment by nlundbo** [comment]
:+1: 
a bit confused though. Remember approving a python library before with the same change. But it was not this lib :P

**Review by voroninman — APPROVED** [review]
LGTM. One question about dropping `start_logging`.

File: wrapplog.py (hunk 2 above)
Comment (heinrisch) [reply wrapplog.py]: My thought was force removal, but since its python I'll add this back..

**Pull request merged**